cmd/brew-services.rb

```diff
@@ -236,6 +237,33 @@ def restart(target)
       end
     end
 
+    # "load" a plist
+    def launchctl_load(plist, _function, service)
```

Comment:
> Is it OK to change the output for the brew run case?

Yes.



> (On the other hand, maybe brew run really does start things instead of running them. In which case this should all just go away?)

That was to preserve backwards compatibility of commands.